src/views/components/dockError.ejs

```diff
@@ -0,0 +1,14 @@
+<html>
+  <head>
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <div class="alert alert-danger" role="alert">
+      Stand ID is incorrect
```

Comment:
Maybe not hardcode this error message? Display depending on what response code you're getting. If you also get what the response is for, you can remove the redundant codes for dock error, stand error and success. There only needs to be one response component.